src/ldp/http/SimpleBodyParser.ts

```diff
@@ -1,4 +1,5 @@
 import { BodyParser } from './BodyParser';
+import { DATA_TYPE_QUAD } from '../../util/Constants';
```

Comment:
Probably want to name this file `Datatypes` instead of just constants?

Reply:
One of them is used as a content-type instead of a Representation dataType though

Reply:
What is the difference though, how are either used (I probably need a refresher)?
And since they’re constants anyway, can’t we use the same?

Reply:
Maybe dataType should be an enum though.

Reply:
`dataType` is a field in Representation, representing the contents of the data Readable. The content type is used to request Quads from a ResourceStore (as there is no existing content-type for that such as `text/turtle`).

Reply:
I'm not sure if it would ever be necessary since those 2 options probably cover most of what we will ever need, but making it an enum does make it more difficult should there ever be an external extension with a new dataType.

Reply:
Okay, got it. I wonder if we just want a `binary` flag then, but probably not.

Let's indeed go your way, perhaps with separate constant files `ContentTypes` and `StreamTypes` (and maybe call the field `streamType` then).

Reply:
The field containing the stream is called `data` so `dataType` does make sense I think

Reply:
OK, fair enough. Then perhaps `ContentTypes` as a file will work, and just those exceptions then.